engine/modelc/src/modelimporter_jni.cpp

```diff
@@ -826,70 +840,71 @@ static void CreateSkins(JNIEnv* env, const dmModelImporter::Scene* scene, dmArra
     for (uint32_t i = 0; i < count; ++i)
     {
         Skin* skin = &scene->m_Skins[i];
-        skins[skin->m_Index] = CreateSkin(env, skin);
+        skins[skin->m_Index] = CreateSkin(env, types, skin);
     }
 }
 
 
 static jobject CreateJavaScene(JNIEnv* env, const dmModelImporter::Scene* scene)
 {
-    InitializeJNITypes(env);
+    TypeInfos types;
+    InitializeJNITypes(env, &types);
```

Comment:
Since this call may come from different threads, we need to handle the data accordingly.

The values of the class types actually differ too, so it's not enough to load them when the library is loaded.